src/index.js

```diff
@@ -0,0 +1,196 @@
+import _, { add, times } from 'lodash';
+import { clearAll } from './script.js';
+import taskCompleted from './script.js';
+import './style.css';
+
+const tasks = [
+	{
+		description: 'Go to the GYM',
+		completed: true,
+		index: 0
+	},
+	{
+		description: 'Read ES6 book',
+		completed: true,
+		index: 1
+	},
```

Comment:
- [ ] Please remove all hard-coded tasks. The aim of this milestone is to use user-generated data by implementing add and remove functions.